setup.py

```diff
@@ -35,6 +35,8 @@
            'scripts/yujin_tools_settings',
            'scripts/yujin_list_dirs_by_size',
            'scripts/yujin_list_git_branches',
+           'scripts/rosdep-generator/rosdep-generator',
```

Comment:
I dont think so... It should get the file from inside the dir, and copy it in the same place as other scripts.